src/components/Editor/index.tsx

```diff
@@ -1,4 +1,6 @@
-/* eslint max-lines: ["error", 350] */
+/* eslint max-lines: ["error", 400] */
+/* eslint max-lines-per-function: ["error", 180] */
+/* eslint max-statements: ["error", 25] */
```

Comment:
@coderabbitai submit an issue for refactoring this file so we can get rid of the linter config overrides